python/cac_tripplanner/destinations/views.py

```diff
@@ -45,7 +45,11 @@ def isochrone(self, lat, lng, mode, date, time, max_travel_time, max_walk_distan
         isochrone_response = requests.get(self.isochrone_url, params=payload)
 
         # Parse and traverse JSON from OTP so that we return only geometries
-        json_poly = json.loads(isochrone_response.content)[0]['geometry']['geometries']
+        try:
+            json_poly = json.loads(isochrone_response.content)[0]['geometry']['geometries']
```

Comment:
I haven't looked at the feed parser, but at least here we're using requests https://github.com/azavea/cac-tripplanner/blob/2a90ebd68885efc0549988167223a2c0a45bb6a7/python/cac_tripplanner/destinations/views.py#L45